HW0/P5/P5.txt

```diff
@@ -1 +1,17 @@
 # Your answers here.
+
+1. 255s, because the first two bags are paired and there is an additional second for every subsequent bag.
+
+2. Yes. In the first second, all 8 cashiers sum 2 bags each, for a total of 16 bags. In each subsequent second, each cashier adds an additional bag to their running total, for an additional 8 bags per second. Then, the final 8 sums would have to be added together, which can be done in parallel - log base 2. Therefore, 256 bags will take 1 + ((256-16)/8) + 3 = 34s.
+
+3. Infinite cashiers means that we can assume that we always have the exact number of cashiers we need to sum the bags most efficiently. In the first second, all bags can be converted to pairs that are added together, leaving half the number of numbers to sum as before. This repeats until there is a single sum of all the bags. This is the pattern for log base 2, meaning that the bags can be added in 8s.
+
+4. [See graph. Note that each curve has a distinctly labeled y-axis.]
```

Comment:
This is super confusing. Why not plot both using a log-log scale? That will fix the problem with scale, i.e. you won't need two axes.